themes/bootstrap3/templates/myresearch/odmycontent.phtml

```diff
@@ -64,23 +79,24 @@
             <?php if (count($listAuthors) > 1): ?>, <?=$this->transEsc('more_authors_abbrev')?><?php endif; ?><br>
           <?php endif; ?>
 
-          <div><?=$this->transEsc('od_expires_on', ['%%due_date%%' => $resource['checkout']->expires])?></div>
+          <div> <?=$this->transEsc('od_expires_on', ['%%due_date%%' => $resource['checkout']->expires])?></div>
           <div class="pull-right">
             <?php if ($resource['checkout']->isReturnable): ?>
               <a
-                class="btn btn-primary icon-link returnTitle"
+                class="btn btn-warning icon-link returnTitle"
                 data-lightbox title="<?=$this->transEsc('od_but_return')?>"
-                href="<?=$this->url('overdrive-hold') . "?od_id=$od_id&rec_id=$rec_id&action=returnTitleConfirm"; ?>"
+                href="<?=$this->url('overdrive-hold') . "?od_id=$od_id&rec_id=$rec_id&edition=" .
```

Comment:
Are the various IDs here guaranteed numeric, or should we be urlencoding all of them?

Reply:
As far as I know they are always alphnum but I'll just encode them to be on the safe side.